daemon/database/models/SwapIn.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"time"
+)
+
+type SwapIn struct {
+	ID uint `gorm:"primaryKey;autoIncrement"`
+
+	// The swap identifier for the swap service
+	SwapID      string     `gorm:"not null"`
+	AmountSATS  uint64     `gorm:"not null"`
+	Status      SwapStatus `gorm:"type:status_enum;not null"`
+	SourceChain Chain      `gorm:"type:chain_enum;not null"`
+
+	// The address where we will pay to
+	DestinationAddress string
+	DestinationTx      string
```

Comment:
I do believe now that ClaimTx is a better name, what's this param? the TxId?

Reply:
yes, TxId. Let's rename it to ClaimTx so we are in line with the server

Reply:
done